distr_tracing/distr_tracing_rn/distr-tracing-rn-3-1.adoc

```diff
@@ -0,0 +1,108 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="distributed-tracing-rn-3-1"]
+= Release notes for {DTProductName} 3.1
+:context: distributed-tracing-rn-3-1
+
+toc::[]
+
+include::modules/distr-tracing-product-overview.adoc[leveloffset=+1]
+
+[id="distributed-tracing-rn_3-1_component-versions"]
+== Component versions in the {DTProductName} 3.1
+
+[options="header"]
+|===
+|Operator |Component |Version
+
+|{TempoName}
+|Tempo
+|2.3.1
+
+//for each new release, update the release number in the xref path on the following line
+|xref:../../otel/otel_rn/otel-rn-3.1.adoc[{OTELName}]
+|OpenTelemetry
+|0.93.0
+
+|{JaegerName} (deprecated)
+|Jaeger
+|1.53.0
+
+|===
+
+// Tempo section
+[id="distributed-tracing-rn_3-1_tempo-release-notes"]
+== {TempoName}
+
+[id="distributed-tracing-rn_3-1_tempo-release-notes_new-features-and-enhancements"]
+=== New features and enhancements
+
+This update introduces the following enhancements for the {TempoShortName}:
+
+* Monolithic custom resource in dev-preview support.
+* Support for cluster-wide proxy environments.
+* Support for TraceQL to Gateway component.
+
+[id="distributed-tracing-rn_3-1_tempo-release-notes_bug-fixes"]
+=== Bug fixes
+
+This update introduces the following bug fixes for the {TempoShortName}:
+
+* Before this update, when a TempoStack was created with the `monitorTab` enabled in {product-title} 4.15, the required `tempo-redmetrics-cluster-monitoring-view` ClusterRoleBinding was not created. This update resolves the issue by fixing the operator RBAC for the monitor tab when the operator is deployed in an arbitrary namespace. (link:https://issues.redhat.com/browse/TRACING-3786[TRACING-3786])
```

Comment:
```suggestion
* Before this update, when a `TempoStack` object was created with the `monitorTab` field enabled in {product-title} 4.15, the required `tempo-redmetrics-cluster-monitoring-view` cluster role binding was not created. This update resolves the issue by fixing the Operator RBAC for the monitor tab when the Operator is deployed in an arbitrary namespace. (link:https://issues.redhat.com/browse/TRACING-3786[TRACING-3786])
```
Operator must always be capitalized: https://redhat-documentation.github.io/supplementary-style-guide/#operator

Reply:
Thank you. I'm pretty sure I was advised to use lowercase during one of previous PR reviews for the OpenShift docs.